File: 3. IRRATATING PARROT[1].py
#IRRATATING PARROT[1]
while True:
    name = input("Please enter name: ")
    print("Your name is %s." %name)

    age = input("Please enter age: ")
    print("Your age is %s." %age)

    question = input("Please enter the correct word to brake out of the never ending loop: ")
    if question == "word":
        break
    else:
        print("You are a failure.")

File: Python Workbook in C++/3. IRRATATING PARROT[1]/3. IRRATATING PARROT[1]/3. IRRATATING PARROT[1].cpp
// 3. IRRATATING PARROT[1].cpp : Defines the entry point for the console application.
//

#include "stdafx.h"
#include <iostream>
#include <string>

using namespace std;

int main()
{
	string name, question;
	unsigned int age;
	while (true) {
		cout << "Please enter name: ";
		cin >> name;
		cout << "Your name is " << name << endl;

		cout << "Please enter age: ";
		cin >> age;
		cout << "Your age is " << age << endl;

		cout << "Please enter the correct word to brake out of the never ending loop: ";
		cin >> question;
		
		if (question == "word") {
			break;
		}
		else {
			cout << "You are a failure." << endl;
		}
	}
    return 0;
}

